bevy_asset_loader/examples/progress_tracking.rs

```diff
@@ -15,21 +15,25 @@ fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
         .add_state::<MyStates>()
+        .init_resource::<iyes_progress::ProgressCounter>()
```

Comment:
Should this not happen in iyes_progress?

Reply:
Yes, I am not sure how that ended up there, sorry. Removed it in the last commit